Make capsule protocol usage more explicit (#120)
ietf-wg-webtrans/draft-ietf-webtrans-http3: draft-ietf-webtrans-http3.md

error.

The `webtransport` HTTP Upgrade Token uses the Capsule Protocol as defined in
{{HTTP-DATAGRAM}}.  The Capsule Protocol is negotiated when the server sends a
2xx response.  The `capsule-protocol` header field {{Section 3.4 of

> HTTP-DATAGRAM}} is not used in WebTransport and MUST be ignored by WebTransport

Reviewer: As written this discourages the use of the header. Here's a proposal that avoids that issue:

```suggestion
HTTP-DATAGRAM}} is not required by WebTransport and can safely be ignored by WebTransport
```
